fixham/go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/projectcalico/fixham
```

Comment:
fixham is a separate module that can be use on its own similar to api

Reply:
I believe since we're narrowing in on release logic, we don't need to treat fixham as a library to be imported, rather it's a CLI tool that will orchestrate the release. So with that in mind, I think it will be better to use the existing go.mod / go.sum from the repository root.